persistence/src/androidMain/kotlin/com/wire/kalium/persistence/db/GlobalDatabaseProvider.kt

```diff
@@ -30,18 +30,21 @@ actual class GlobalDatabaseProvider(
     private val database: GlobalDatabase
 
     init {
+        val schema = GlobalDatabase.Schema
         driver = if (encrypt) {
+            System.loadLibrary("sqlcipher")
             AndroidSqliteDriver(
-                schema = GlobalDatabase.Schema,
+                schema = schema,
                 context = context,
                 name = dbName,
-                factory = SupportFactory(passphrase.value)
+                factory = SupportOpenHelperFactory(passphrase.value, null, true)
```

Comment:
As this is a Java method, we can't pass named arguments.

The `true` here is for `enableWriteAheadLogging`.

Reply:
maybe define a variable and pass that variable?

Reply:
I think it's a bit too much, and still not optimal (_i.e._ we'd still be able to put the right variable in the wrong place without noticing).


IntelliJ suggests adding something like this:
```kt
SupportOpenHelperFactory(
    /** password **/ passphrase.value,
    /** enableWriteAheadLogging */ true
)
```

Which I don't hate, just find ugly 😅 

I'm not convinced by any approach, I dislike them all equally 🤣 

But I'll keep an eye open for the next one.